politeiawww/cmd/politeiawwwcli/client/client.go

```diff
@@ -379,6 +379,52 @@ func (c *Client) Policy() (*v1.PolicyReply, error) {
 	return &pr, nil
```

Comment:
This probably should be its own tool as well where the generic code is shared. Splitting that can happen after CMS is up though.